### PR TITLE
Update for prometheus-community

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

**Review by SuperQ — CHANGES_REQUESTED** [review]

File: cmd/avalanche.go [hunk 1]
```diff
@@ -36,7 +52,7 @@ var (
 )
 
 func main() {
-	kingpin.Version("0.3")
+	kingpin.Version(version.Print("node_exporter"))
```
[the remark this answers is not in the document]
Comment (roidelapluie) [reply cmd/avalanche.go]: updated

File: cmd/avalanche.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2022 The Prometheus Authors
+// Copyright 2018 FreshTracks
```
[the remark this answers is not in the document]
Comment (roidelapluie) [reply cmd/avalanche.go]: updated

**Comment by roidelapluie** [comment]
Any idea why circle ci does not run?

**Comment by SuperQ** [comment]
Enabled "build fork pull requests" in CircleCI.

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**